spec.md

```diff
@@ -1125,6 +1141,11 @@ message ControllerServiceCapability {
       // with the snapshot_id as the filter to query whether the
       // uploading process is complete or not.
       LIST_SNAPSHOTS = 6;
+      // Plugins supporting efficient volume cloning MAY report this
+      // capability.  Source volume must be managed by the same plugin.
+      // Not all volume sources and parameters combinations MAY work.
```

Comment:
MAY indicates a recommendation, or "soft" requirement - does it make sense in this context? or would "may" (non-RFC verbage) suffice?

Reply:
Normal may will suffice in my opinion.  Changing it.